### ISSUE TITLE
Has development grown stale?

### ISSUE THREAD

**Issue by maurice-schleussinger** [oblador/hush#251]
It feels like more and more cookie popups are not hidden in a way that I can still use the website. The popups are hidden (most of the time), but often the page cannot be used (e.g. no scrolling).

Is there a technical reason for this or has development for this extension stopped?

**Comment by USMC03Ronin** [comment]
Yes no responses from dev look at app reviews on apple app store, no replies

**Comment by lukelbro** [comment]
From the owner of the repo’s profile:

> [My repos](https://oblador.github.io/) demand more time than I have, so I have to be ruthless in my prioritization. Send me a DM on [Twitter](https://twitter.com/trastknast) if you want to help out.

I’m not ready to say it is abandoned just yet. The last App Store update was 5 months ago, as was the last commit. For someone who is very busy that seems reasonable. Although with the rate at which the internet changes it is probably unfair to expect functionality to be maintained unless more people contribute. 

it you have any alternatives that you think are better then it would be great if you could share them.

**Comment by USMC03Ronin** [comment]
Thanks for the reply and honesty.

**Comment by oblador** [comment]
Uploaded a new build that hopefully will be approved soon :-)
